src/portable/ReactUi/react/src/components/NavPanel.tsx

```diff
@@ -20,12 +20,13 @@ class NavPanel extends React.Component<IProps, object> {
         const user = users.filter(u => u.username.id === selectedUser)[0];
         const admin = user && user.role && user.role.filter(r => r === "administrator")[0];
         const project = tasks.filter(t => t.id === selectedProject)[0];
+        const backLinkWrapper = admin !== null && tasks.length === 1 && users.length === 1? "":<BackLink target="/" />
```

Comment:
so what happens if admin is undefined? Using admin != null will make fail if admin is undefined or null whereas I think this code will throw an error if admin is undefined.